src-input/duk_bi_date.c

```diff
@@ -958,7 +958,7 @@ DUK_LOCAL duk_ret_t duk__set_this_timeval_from_dparts(duk_hthread *thr, duk_doub
 }
 
 /* 'out_buf' must be at least DUK_BI_DATE_ISO8601_BUFSIZE long. */
```

Comment:
This leftover comment may be misleading since the actual size is being passed now.